Sources/AppDelegate.swift

```diff
@@ -9,16 +9,62 @@ import Cocoa
 import SwiftSoup
 import Then
 
+// MARK: Contribute Data Class
+
+class ContributeData{
```

Comment:
```suggestion 
class ContributeData {
```
공백 하나 추가되면 좋을 것 같습니다.